packages/components/addon/components/hds/side-nav/section/item.hbs

```diff
@@ -0,0 +1,27 @@
+<li class="hds-side-nav__list-item">
+  <Hds::Interactive
+    @href={{@href}}
+    @route={{@route}}
+    @models={{hds-link-to-models @model @models}}
+    @query={{hds-link-to-query @query}}
+    @icon={{@icon}}
+    @isRouteExternal={{@isRouteExternal}}
+    @current-when={{@current-when}}
+    class="hds-side-nav__link hds-typography-body-200 hds-font-weight-medium"
+    ...attributes
+  >
+    {{#if @icon}}
+      <FlightIcon class="hds-side-nav__list-icon" @name={{@icon}} @stretched={{true}} />
+    {{/if}}
+
+    {{@label}}
+
+    {{#if @badge}}
+      <Hds::Badge @text={{@badge}} @type="inverted" @size="small" />
+    {{/if}}
+
+    {{#if @count}}
```

Comment:
according to [the design specifications](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/branch/kGo6tmw1Jk4M94x7qJd4FV/HDS-Product---Components?node-id=27214%3A64617&t=V4bjvEgUYsqXqFAX-1), the badge count should come before the normal badge: 
<img width="791" alt="screenshot_2349" src="https://user-images.githubusercontent.com/686239/217513326-a3b306a1-eb2d-463b-b5ff-26e33367393c.png">

Reply:
Ok I'll change it